src/TArea.h

```diff
@@ -124,8 +124,4 @@ class TArea
     // Made private as we may change implementation detail
 };
 
-// - gezeichnet werden erstmal die areas
-//   unter berücksichtigung der an die area angrenzenden edges
-// - der span der area ist unterschiedlich
```

Comment:
The areas are drawn first
Taking into account the edges adjacent to the area
The span of the area is different

Reply:
We do have a pair of long unused members (introduced in 41e0cbbed58f518321328e236429ce4d7097c6d6) in the TArea class that might relate to this - see the header file:
```cpp
    QSet<int> rooms; // rooms of this area
    // TODO: These next 2 members have not been used for some time - if at all
    // - maybe they can go?
    QVector3D pos;   // pos auf der map und 0 punkt des area internen koordinatensystems
    QVector3D span;
    int min_x;
    int min_y;
    int min_z;
    int max_x;
    int max_y;
    int max_z;
```

Reply:
That comment translates to

Position on the map and point zero of the area's internal coordinate system